tests/test-definition.yaml

```diff
@@ -5,11 +5,12 @@ dimensions:
       - 3.3.4
       - 3.3.6
       - 3.4.0
+      - 3.4.1
       # To use a custom image, add a comma and the full name after the product version
       # - 3.4.0,oci.stackable.tech/sandbox/hadoop:3.4.0-stackable0.0.0-dev
   - name: hadoop-latest
     values:
-      - 3.4.0
+      - 3.4.1
       # To use a custom image, add a comma and the full name after the product version
       # - 3.4.0,oci.stackable.tech/sandbox/hadoop:3.4.0-stackable0.0.0-dev
```

Comment:
suggestion: Update the custom image version to 3.4.1 as well.

Reply:
done https://github.com/stackabletech/hdfs-operator/pull/656/commits/eba40eb1e8ecba90676870823a6bc3a8d1031f65